lte/gateway/docker/deploy/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN mkdir -p ~/.ssh &&  \
     chmod 600 id_rsa
 
 ARG CACHE_DATE=2019-09-05
```

Comment:
_optional (because probably out of scope):_
Candidate for removal, as not used anywhere, right?

Reply:
out of scope

Also: even seemingly unused args trigger a rebuild if they are changed and that is the usecase here. So you can use that Argument to force rebuild of the cached layers.